guide/src/chapter/reference_guide/command_line_interface/README.md

```diff
@@ -0,0 +1,78 @@
+# Command
+
+The main command is `punktf`.
+
+```text,ignore
```

Comment:
now we have to update this manually :/

Reply:
This is directly taken from the wiki so we already had that :sweat_smile:.

My goal is to remove the help text, as like you say, it can just be gotten by the `--help` option.
What i want to do though, is to have documentation for each command and describe what the options do.